Casks/aurora-hdr.rb

```diff
@@ -0,0 +1,18 @@
+cask 'aurora-hdr' do
+  version :latest
+  sha256 :no_check # required as upstream package is updated in-place
+
+  url 'https://downloads.skylum.com/aurorahdr2019/mac/AuroraHDR2019Installer.zip'
+  appcast 'http://aurorahdr2019mac.update.skylum.com'
+  name 'AuroraHDR'
```

Comment:
```suggestion
  name 'Aurora HDR'
```

(with a space)

Also, please add the original Japanese name in a new `name` underneath.

Reply:
I think this product has the English name as described.

Reply:
Yes, but we’d like to have both.

Reply:
I'm sorry. There was a shortage in my explanation.
There is no Japanese name for this software.